content/docs/user-guide/remote-storage.md

```diff
@@ -0,0 +1,200 @@
+remote storage. Locally, only the most recent three versions of the project are
+stored. The data scientist recently cleared the local cache using DVC's garbage
+collection command `dvc gc`. Periodically, she/he issues `dvc push` to send new
+committed versions of the project to remote storage, and can retrieve old
+versions using `dvc pull` if needed.
+
+## Connecting and Pushing to Remote Storage
+
+Multiple cloud storage providers can be used with DVC, and connecting is fairly
+straightforward using the `dvc remote` command. For example, to connect your
+project to an [Amazon AWS](https://aws.amazon.com) S3 bucket, name the remote
+`s3remote`, and set it as your default remote storage, you would issue the
+following command:
+
+```dvc
+$ dvc remote add --default s3remote s3://path/to/cache
+```
+
+The `dvc remote add` docs outline connecting DVC to other specific providers
+such as [Microsoft Azure](https://azure.microsoft.com/), and we've also provided
+an example specifically about
+[Google Drive as a DVC remote](/doc/user-guide/setup-google-drive-remote). Once
+your remotes are configured, you can double check that the remote storage was
+added correctly using `dvc remote list`. In this example, two remotes are set
+up, one on the cloud and one on a `work` folder on a mounted volume:
+
+```dvc
+$ dvc remote list
+s3cache	s3://yourbucket/yourremotecache/
+work  	/Volumes/ds_team/work
+```
+
+💡 Before adding cloud remote storage to your project, you need to ensure that
+you have configured access to the remote storage correctly. This is dependent on
+how your cloud storage provider works and will vary depending on the provider.
+For example, for if you want to use an Amazon S3 bucket for remote storage then
+you need to
+[configure DVC for S3 access](/doc/command-reference/remote/add#supported-storage-types)
+and [install the S3 version of DVC](/doc/install). Further instructions are
+specific to the cloud service providers and may change from time to time.
+
+One remote storage is correctly set up, there are three main steps you need to
+take store versions of your project there:
+
+1. Commit the project to your local cache with `dvc commit`; this makes a new
+   local version of the project.
+2. Push this new version to the remote storage with `dvc push` (use the `-r`
+   option to specify which remote storage if you are not using the default
+   storage).
+3. Optionally clear out your local workstation cache with `dvc gc`, the garbage
+   collector, to free up space on the local filesystem.
+
+A local folder can also be used as remote storage as shown in the example above
+where a remote storage named `work` has been configured. This is useful if, for
+example, you have a network share mounted and want to use that instead of a
+cloud provider.
+
+## Content Addressable Storage Format
+
+DVC optimises the storage space used in the local cache and remotes by ensuring
```

Comment:
Thanks for this section, it's definitely interesting content as I mentioned in the previous PR but still this is more about the DVC cache mechanism. Please make it into a separate page somewhere and leave it out of this doc or link to if if/where needed to make the review process easier.